lib/hooks/useAwardsList.js

```diff
@@ -37,26 +37,50 @@ export const useAwardsList = () => {
     setPrizeEstimate(estimatedPoolPrize)
   }, [poolTotalSupply, supplyRatePerBlock, prizePeriodRemainingSeconds, awardBalance])
 
-  if (erc20Awards.loading) {
-    return {
-      loading: true,
-      awards: []
+  const awards = useMemo(() => {
+    if (erc20Awards.loading) {
+      return []
     }
-  }
 
-  let awards = [...erc20Awards.awards]
+    let awards = [...erc20Awards.awards]
+
+    if (poolChainValues.sablierStream) {
+      const formattedBalance = numberWithCommas(poolChainValues.sablierStream.amountUnformatted, {
+        decimals: poolChainValues.sablierStream.tokenDecimals
```

Comment:
This feels like it could be refactored a bit. something like:

```
const { sablierStream } = poolChainValues
const {
  tokenName,
  tokenSymbol,
  tokenAddress,
  amountUnformatted,
  tokenDecimals,
  amountUnformatted
} = sablierStream
...
```

```
if (sablierStream) {
  awards.unshift(_streamAwardToken()) // new function
}
```

}